### ISSUE TITLE
Couldn't factory reset on 0.14, but could on 0.12-git - Module not found

### ISSUE THREAD

**Issue by LSgeo** [uunicorn/python-validity#149]
Hi, I was trying to enroll fingerprints on Arch on a Lenovo T480s, using aur/python-validity 0.14-1 installed using paru. `fprintd-enroll` was returning `list_devices failed:`, and I was attempting to [follow instructions](https://github.com/uunicorn/python-validity/issues/3#issuecomment-985283517) to factory reset the fingerprint reader. However, attempting to do so returned a python module not found error. 

I installed the git version available, aur/python-validity-git 0.12.r0.gfcd862d-1, and could successfully factory reset without the module not found error. Since the non-git version has a higher version number, I wonder if this is a bug introduced between 012.r0 and 0.14-1?

Unfortunately, given the extensive paru output filling my terminal scrollback, I've lost the exact module - but I believe it was `validitysensor`. I couldn't reproduce the error by attempting to recreate it with the original process.

Long story short, I managed to reset and enroll my fingerprint using 0.12-git, but not 0.14 from AUR.
